app/controllers/UploadController.scala

```diff
@@ -63,7 +63,7 @@ class UploadController @Inject()(val controllerComponents: SecurityComponents,
         case Some("InProgress") =>
           Ok(views.html.uploadInProgress(consignmentId, pageHeading, request.token.name)).uncache()
         case Some("Completed") =>
-          Ok(views.html.uploadHasCompleted(consignmentId, pageHeading, request.token.name)).uncache()
+          Ok(views.html.uploadHasCompleted(consignmentId, pageHeading, request.token.name, request.token.isJudgmentUser)).uncache()
```

Comment:
I'm just wondering whether it's best to use `isJudgmentUser` here. This action is wrapped with the `judgmentTypeAction` which means that if the consignment type is not set to judgment, then they get forbidden. What could happen, it's unlikely I know is:
* A user logs in as a judgment user and creates a consignment.
* The user later logs in as a standard user and goes to this consignment.
* They will see the judgment upload page because of the consignment type but the link will be pointing to the standard user processing page because of their token claim.

It might be safer to just put `true` here and `false` on line 41 (as that page can only be accessed by standard consignment types. 

I'm not 100% sure on this though, what do you think? There's other places where we're doing `request.token.isJudgmentUser` inside actions wrapped in `judgmentTypeAction` so if you think it's worth changing here then we can change those as well.

Reply:
Ah, so this is the case where the user is both a judgment user and standard user? Then `request.token.isJudgmentUser` would return `true` in both cases? Yes, it makes sense then.